src/Bart/Jenkins/Job.php

```diff
@@ -9,57 +8,72 @@
  */
 class Job
 {
-	private $base_job_url;
-	private $default_params = array();
-	private $my_build_id;
+	private $defaultParams = [];
+	private $myBuildId;
 	private $metadata;
 	/** @var \Logger */
 	private $logger;
 
+	/** @var Connection $connection */
+	private $connection;
+	/** @var string $baseApiPath */
+	private $baseApiPath;
+
 	/**
-	 * Load metadata about a project
-	 */
-	public function __construct($domain, $job_name)
+	 * Job constructor. Loads metadata about a project.
+	 * @param Connection $connection
+	 * @param array $projects This parameter specifies the location of the Jenkins Job.
```

Comment:
Yeah, I have gone back and forth on this too. My thinking was that if we require the path, the user will have to pass `job/ExampleProjectBase/job/ExampleProjectSub`, as opposed to just passing the projects list as an array, and the `Job` class can handle adding the `job/` where appropriate. But I think accepting the path might be the most appropriate. My first thought was to use a `LinkedList`.